src/App.tsx

```diff
@@ -1,14 +1,67 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [selectedTodoId, setSelectedTodoId] = useState(0);
+  const [status, setStatus] = useState('all');
+  const [query, setQuery] = useState('');
+  let visibleTodos = todosFromServer;
+  const todoToModal = visibleTodos.filter(todo => todo.id === selectedTodoId);
+
+  useEffect(() => {
+    const getTodosFromServer = async () => {
+      const todos = await getTodos();
+
+      setTodosFromServer(todos);
+    };
+
+    getTodosFromServer()
+      .catch(Error);
```

Comment:
use try catch
let us create separate file like `helper` for function and import them where it needed